ui/apps/platform/cypress/integration/vulnerabilities/VulnerabilityReporting/vulnerabilityReportingNav.test.ts

```diff
@@ -0,0 +1,104 @@
+import withAuth from '../../../helpers/basicAuth';
+import {
+    visitVulnerabilityReports,
+    vulnerabilityReportsBasePath,
+} from './VulnerabilityReporting.helpers';
+
+const staticResponseMap = {
+    'reports/configurations': {
+        fixture: 'vulnerabilities/vulnerabilityReporting/vulnerabilityReports.json',
+    },
+};
+
+function interceptVulnerabilityReport(reportId, response, interaction) {
+    cy.intercept('GET', `/v2/reports/configurations/${reportId}`, response).as('reportDetails');
+
+    interaction();
+    cy.wait('@reportDetails');
+}
+
+describe('Vulnerability Reporting Navigation', () => {
+    withAuth();
+
+    it('navigates to the index page', () => {
+        visitVulnerabilityReports(staticResponseMap);
+        cy.url().should('include', vulnerabilityReportsBasePath);
+        cy.get('h1').should('contain', 'Vulnerability reporting');
+    });
+
+    it('navigates to the create page', () => {
+        visitVulnerabilityReports(staticResponseMap);
+        cy.contains('button', 'Create report').click();
+        cy.url().should('include', `${vulnerabilityReportsBasePath}?action=create`);
+        cy.get('h1').should('contain', 'Create report');
+    });
+
+    it('navigates to the details page', () => {
+        visitVulnerabilityReports(staticResponseMap).then(([{ response }]) => {
+            const firstReport = response.body.reportConfigs[0];
+
+            interceptVulnerabilityReport(firstReport.id, firstReport, () => {
+                cy.contains('a', firstReport.name).click();
```

Comment:
If this is link in table cell, what are pro and con of `cy.get('td').contains(…).click()`

https://docs.cypress.io/api/commands/contains

Reply:
I think that would be an improvement. More obvious whats happening and less prone to a "misclick".